doc/specs/vulkan/chapters/memory.txt

```diff
@@ -1721,6 +1721,7 @@ ifdef::VK_KHX_device_group[]
   * [[VUID-vkMapMemory-memory-00683]]
     pname:memory must: not have been allocated with multiple instances.
 endif::VK_KHX_device_group[]
+  * pname:ppData must: be a valid pointer to a pointer
```

Comment:
autogenerated original would be:
> pname:ppData must: be a pointer to a pointer

autogenerated new version would be:
> pname:ppData must: be a valid pointer to a valid pointer

Both would be lies.
As the manual words say, the outer pointer does have to be "valid" the inner does not.

Autogenerated words aside, it is a matter of semantics of the `vk.xml`. Autovalidity would imply both the outer and inner pointer have to be "valid" in some sense. You are silently relying on `void*` with implicit `len==1` to be valid by some too smartistic fashion (little comparable to SFINAE).
Which fails, because elsewhere you make a different semantic for `void*`, that it is "a pointer to bytes" and **not** "pointer to nothing", e.g. `VkPipelineCacheCreateInfo`:

> Valid Usage (Implicit)
>------------------------
> If `initialDataSize` is not `0`, `pInitialData` **must** be a pointer to an array of `initialDataSize` bytes